.github/workflows/unity-version-badge.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           git config --global user.name 'github-actions'
           git config --global user.email 'github-actions@github.com'
           git add README.md
-          git add etc/PackageTemplate/README.md
```

Comment:
I think this is supposed to be in a different PR?

Reply:
Yep! This change is actually also in a PR that is opened [here](https://github.com/PlayEveryWare/eos_plugin_for_unity/pull/786). 

The reason I merged it into this PR is because _this_ PR triggers the workflow file that has been changed, and GitHub's user interface displays a "failed check" for the PR because it is broken in `development`, so I pre-emptively merged it into this PR to avoid the confusion that such a warning would generate.

Reply:
As an example - _any_ PR opened into `development` after the merge that moved the `README.md` file into `com.playeveryware.eos` will have a failed check. It just so happens that this PR is the first one to trigger it.